rnd/autogpt_builder/.prettierrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Let's work with Prettier's defaults for now. No need for this file.

Reply:
Sounds good, I'll keep it to the defaults suggested on their site:
[Configuration File](https://prettier.io/docs/en/configuration.html)

Reply:
Those are not the defaults. Please just delete the file.

Reply:
Can do. I have removed the file.